src/nibiru/index.ts

```diff
@@ -52,12 +23,13 @@ export const getPlaynets = async (): Promise<LocalConfig[] | undefined> => {
 export const getNibiruChains = async (): Promise<{
   [key: string]: LocalConfig;
 }> => {
-  const [playnets, devnets, itn] = await Promise.all([
+  const [playnets, devnets, itn, main] = await Promise.all([
     undefined, //getPlaynets(),
```

Comment:
You can get rid of playnet and devnet, we'll likely never expose these on our explorer.